targets/TARGET_STM/stm_spi_api.c

```diff
@@ -90,7 +90,19 @@ void spi_get_capabilities(PinName ssel, bool slave, spi_capabilities_t *cap)
     } else {
         cap->minimum_frequency = 200000;          // 200 kHz
         cap->maximum_frequency = 2000000;         // 2 MHz
+#if defined(TARGET_STM32F0) || defined(TARGET_STM32F3) || defined(TARGET_STM32F7) || \
+    defined(TARGET_STM32G0) || defined(TARGET_STM32G4) || defined(TARGET_STM32L4) || \
+    defined(TARGET_STM32L5) || defined(TARGET_STM32WB)
+        cap->word_length = 0x0000FFF8;            // 4 through 16 bit symbols, inclusive
+#elif defined(TARGET_STM32H7)
+        cap->word_length = 0xFFFFFFF8;            // 4 through 32 bit symbols, inclusive
+#elif defined(TARGET_STM32F1) || defined(TARGET_STM32F2) || defined(TARGET_STM32F4) || \
+    defined(TARGET_STM32L0) || defined(TARGET_STM32L1)
```

Comment:
Maybe value could be defined in each subfamily in each TARGET_STM32xx/spi_device.h for ex ?

Reply:
I would not mind necessarily. However, there is some precedence for putting specific feature items in here. For example on line 332, you find this: `#if defined SPI1_BASE`, which is then followed by function calls. I understand that this is different from a value, on the other hand, the value is only used here.

I am neutral on this though.